includes/amp-functions.php

```diff
@@ -11,35 +11,41 @@ function amp_add_author_after_content() {
 
     global $post;
 
-    $author_bio        = trim( get_field( 'post_author_bio', $post ) );
-    $author_title      = get_field( 'post_author_title', $post );
-    $author_photo_data = get_field( 'post_author_photo', $post );
-    $author_photo      = $author_photo_data['sizes']['medium'] ?? null; 
-    $author_name       = "By " . get_the_author_meta( 'display_name', $post->post_author );
-    $author_byline     = get_field( 'post_author_byline', $post );
-
-	if ( $author_byline ) {
-		$author_name = $author_byline;
-	}
-
-    ob_start();       
+    $author_data = today_get_post_author_data( $post );
+    $author_byline     = $author_data['name'] ?? null;
+
+    ob_start();      
+	if ( $author_byline ) :
+	    $author_bio  = $author_data['bio'] ?? null;
```

Comment:
Tabbing is a bit wonky in this file--make sure literal tabs are used, not spaces